package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "alby",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "pretty": "prettier --write .",
+    "prepare": "husky install"
+  },
+  "dependencies": {
+    "next": "12.3.1",
```

Comment:
Do we prefer `^` for versions here?

Reply:
On `create next app`, it came as default without `^`. Ig, we can go with the default, right? Let me know what you think.

Reply:
We have a `yarn.lock` to fix deps. Let's add the `^`. But it's non-blocking as well.